flycheck.el

```diff
@@ -7121,8 +7121,8 @@ the `--severity' option to Perl Critic."
 See URL `https://metacpan.org/pod/Perl::Critic'."
   :command ("perlcritic" "--no-color" "--verbose" "%f/%l/%c/%s/%p/%m (%e)\n"
             (option "--severity" flycheck-perlcritic-severity nil
-                    flycheck-option-int)
-            source)
+                    flycheck-option-int))
+  :standard-input t
   :error-patterns
   ((info line-start
          (file-name) "/" line "/" column "/" (any "1") "/"
```

Comment:
What about the `(file-name)` here?  I think it's unlikely that it's going to be correct when it reads from standard input.